### PR TITLE
Fruit icon

### PR DESCRIPTION
**Pull request by Harshal662**
Closes: #31 

added fruit icons in front of their names.

### CODE REVIEW

File: shell/grep_example.txt
Comment (tungbq) [inline shell/grep_example.txt:1]: This should be part of another PR not this PR, could you please double check?

**Review by tungbq — CHANGES_REQUESTED** [review]
Please check my comment

File: shell/grep_example.txt
Comment (Harshal662) [reply shell/grep_example.txt:1]: so you are saying I should only edit individual parts. And the changes made in previous branch should not be present in this branch, right?

Comment (tungbq) [reply shell/grep_example.txt:1]: Yes, that's correct.
Let's create separated branch (e.g: task-1, task-2, task-. Then create separated PR for each task. Which makes us more easier to review and develop.
